packages/shared/src/components/FeedLayout.tsx

```diff
@@ -81,18 +82,78 @@ const baseFeedSettings: FeedContextData[] = [
   },
 ];
 
+const sidebarOpenFeedSettings: FeedContextData[] = [
```

Comment:
Is there any correlation between these settings and the original ones?
If so maybe we can code it. It looks like every element here is `i-1` in the previous array

Reply:
The page size is a bit random sometimes 2, sometimes 3